com.unity.netcode.gameobjects/Editor/NetworkTransformEditor.cs

```diff
@@ -111,6 +113,7 @@ public override void OnInspectorGUI()
             EditorGUILayout.LabelField("Configurations", EditorStyles.boldLabel);
             EditorGUILayout.PropertyField(m_InLocalSpaceProperty);
             EditorGUILayout.PropertyField(m_InterpolateProperty);
+            EditorGUILayout.PropertyField(m_UseFixedUpdateProperty);
```

Comment:
Do you have a specific use case in mind where a user would want to use this? I couldn't really think of a good example which is widely used. Maybe we could keep this as just a property on NetworkTransform and not expose it in the inspector?

Reply:
+1
also the simplest question: what is this property?
is this a toggle to say do interpolation update on Update() vs FixedUpdate()?

Reply:
After investigation, I figured it wasn't needed anymore with my latest changes. I'll remove it